doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc

```diff
@@ -0,0 +1,10 @@
+[id='asynchronous-signal-events-con']
+
+= Asynchronous signal events
+
+In some cases where several process instances from different process definitions are waiting for the same signal. They are generally executed sequentially in the same single thread. But, if one of those process instances throws a runtime exception, all the other process instances are affecte and usually result in a rolled back transaction. To avoid this situation, {PRODUCT} supports using asynchronous signals events for:
+
+* Throwing intermediate signal events
+* End events
+
+In {CENTRAL}, set the *Data Input* value of the throw event to *async* to automatically set the executor service on each session. This ensures that each process instance is signaled in a different transaction.
```

Comment:
Please provide more info here:
* changes are supposed to be made in Process Designer
* the procedure is done in the "Data assignments" section of Properties panel
* Add a data input property with name 'async', other values leave blank.

Reply:
Done